v2/sourcedb-to-spanner/src/test/resources/SourceDbToSpannerLT/mysql-spanner-schema.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE IF NOT EXISTS table1 (
```

Comment:
If we are testing for wide rows, please set the fetchsize as per [user-guide](https://docs.google.com/document/d/1P2uZLPmw_0av1KN4vt7voaxDw4r119SIgKfxSn5vAmE/edit?tab=t.0#bookmark=id.vgljjjz9hidk)

Reply:
Done.